src/app/components/memberInfoForm/memberInfoFormContainer.js

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
+
+import { deleteMember } from '../../actions/teamActions';
+
+import MemberInfoForm from './memberInfoFormComponent';
+
+const mapStateToProps = ({ team }) => ({ editting: team.editting });
+
+const mapDispatchToProps = dispatch => ({ deleteMember: member => dispatch(deleteMember(member)) });
```

Comment:
looks like `deleteMember` doesn't take a `member` arg anymore, so this can be cleaned up.